luma/core/emulator.py

```diff
@@ -198,8 +198,11 @@ class transformer(object):
     """
     def __init__(self, pygame, width, height, scale):
         self._pygame = pygame
+        self._input_size = (width, height)
         self._output_size = (width * scale, height * scale)
-        self.scale = scale
+        self._scale = scale
+        self._led_on = self._pygame.image.load(os.path.join(os.path.dirname(__file__), "images", "led_on.png"))
```

Comment:
sure - lets implement a flake8 policy for this and enforce it though? - as another issue / PR though, as I think there will be some widescale reformatting needed